guides/development-workflow.md

```diff
@@ -0,0 +1,341 @@
+# Development workflow
+
+## Introduction
+
+<!-- Short intro. Link to playbook for the details -->
+
+### Development principles
+
+<!-- Explain where this principles came from and how they change -->
+
+#### Communicate frequently
+
+  - space to learn from each other
+  - make shared decisions
+  - prevent duplication
+  - solve problems quicker
```

Comment:
Lists should also start with a capital letter, if we're going to follow our own [style guide](https://github.com/dxw/playbook/blob/feature/guide-for-devs/guides/style-guide.md).